packages/deploy-tooling/src/cli/index.ts

```diff
@@ -46,6 +46,7 @@ export function createCommand(name: 'deploy' | 'undeploy'): Command {
         .option('--transport <transport-request>', 'Transport number to record the change in the ABAP system')
         .option('--name <bsp-name>', 'Project name of the app')
         .option('--strict-ssl', 'Perform certificate validation (use --no-strict-ssl to deactivate it)')
+        .option('--no-strict-ssl', 'Deactivate certificate validation')
```

Comment:
Do we really need both, `--strict-ssl` and `--no-strict-ssl`? 
Should not one be the default and the other overwrites it? For security relevant things like this I'd say, strict ssl certificate validation is the default and by using option `--no-strict-ssl` you can deactivate it.